src/dataSources/cloudFirestore/session.js

```diff
@@ -72,6 +72,25 @@ function sessions(dbInstance, logger) {
     return result;
   }
 
+  async function findSessionBySlug(slug) {
+    const docSnap = await sessionsCol
+      .where('sessionSlug', '==', slug.toLowerCase())
+      .where('status', '==', 'ACCEPTED')
```

Comment:
what about the other states? Cancelled, Scheduled?

Reply:
Updated